src/lib/netlist/solver/nld_ms_direct_lu.h

```diff
@@ -259,25 +258,25 @@ void matrix_solver_direct_t<m_N, storage_N>::vsetup(analog_net_t::list_t &nets)
 
 #if 1
 
-	/* Sort in descending order by number of connected matrix voltages.
-	 * The idea is, that for Gauss-Seidel algo the first voltage computed
-	 * depends on the greatest number of previous voltages thus taking into
-	 * account the maximum amount of information.
-	 *
-	 * This actually improves performance on popeye slightly. Average
-	 * GS computations reduce from 2.509 to 2.370
-	 *
-	 * Smallest to largest : 2.613
-	 * Unsorted            : 2.509
-	 * Largest to smallest : 2.370
-	 *
-	 * Sorting as a general matrix pre-conditioning is mentioned in
-	 * literature but I have found no articles about Gauss Seidel.
-	 *
-	 * For Gaussian Elimination however increasing order is better suited.
-	 * FIXME: Even better would be to sort on elements right of the matrix diagonal.
-	 *
-	 */
+	// Sort in descending order by number of connected matrix voltages.
+	// The idea is, that for Gauss-Seidel algo the first voltage computed
+	// depends on the greatest number of previous voltages thus taking into
+	// account the maximum amount of information.
+	//
+	// This actually improves performance on popeye slightly. Average
+	// GS computations reduce from 2.509 to 2.370
+	//
+	// Smallest to largest : 2.613
+	// Unsorted            : 2.509
+	// Largest to smallest : 2.370
+	//
+	// Sorting as a general matrix pre-conditioning is mentioned in
+	// literature but I have found no articles about Gauss Seidel.
+	//
+	// For Gaussian Elimination however increasing order is better suited.
+	// FIXME: Even better would be to sort on elements right of the matrix diagonal.
+	//
+	//
```

Comment:
Does converting a comment spanning multiple lines like this to a sequence of single-line comments really give any benefit?

Reply:
No. I try to be consistent and avoid C-style comments and a search picked this up. These comments should be converted to doxygen format. Since the the solver currently isn't used at all (LU-solver), I prefer to leave it as it is until the solver finds a use.

Reply:
Yeah, but this is a case of forcing consistency where it hinders readability.  Using line comments for every line of a multi-line comment containing prose just makes the comment less readable.

FWIW Doxygen will recognise `/**` as the opening of a documentation comment – this will work, and gives less visual clutter than putting `///` on every line:
```
/**
  \brief The brief description

  Longer description.
  \param [in] name Parameter description
 */
```

(This was carried over to Doxygen from JavaDoc.)

Reply:
> FWIW Doxygen will recognise /** as the opening of a documentation comment – this will work, and gives less visual clutter than putting /// on every line:

I agree. I need to use /// since clang-format destroys markdown tables in doxygen comments. Using single line comments I can fix this by excluding them from formatting using the `CommentPragmas` option. Multi-line comments also may - if they contain code examples - make the scripts for code generation more complex.